instrumentation/apache-elasticjob-3.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/apacheelasticjob/v3_0/ElasticJobTest.java

```diff
@@ -0,0 +1,470 @@
+                  .get(AttributeKey.stringKey("scheduling.apache-elasticjob.job.name")))
+          .isEqualTo("javaHttpJob");
+      assertThat(
+              span.getAttributes()
+                  .get(AttributeKey.longKey("scheduling.apache-elasticjob.sharding.total.count")))
+          .isEqualTo(3L);
+      assertThat(
+              span.getAttributes()
+                  .get(
+                      AttributeKey.stringKey(
+                          "scheduling.apache-elasticjob.sharding.item.parameters")))
+          .isEqualTo("{0=Beijing, 1=Shanghai, 2=Guangzhou}");
+
+      assertThat(
+              span.getAttributes().get(AttributeKey.longKey("scheduling.apache-elasticjob.item")))
+          .isIn(0L, 1L, 2L);
+      assertThat(
+              span.getAttributes()
+                  .get(AttributeKey.stringKey("scheduling.apache-elasticjob.task.id")))
+          .contains("javaHttpJob");
+    }
+
+    Set<Long> shardItems =
+        spans.stream()
+            .map(
+                span ->
+                    span.getAttributes()
+                        .get(AttributeKey.longKey("scheduling.apache-elasticjob.item")))
+            .collect(Collectors.toSet());
+    assertThat(shardItems).containsExactlyInAnyOrder(0L, 1L, 2L);
+  }
+
+  @Test
+  public void testSimpleJob() throws InterruptedException {
+    SynchronizedTestSimpleJob job = new SynchronizedTestSimpleJob(2);
+    ScheduleJobBootstrap bootstrap =
+        new ScheduleJobBootstrap(
+            regCenter,
+            job,
+            JobConfiguration.newBuilder("simpleElasticJob", 2)
+                .cron("0/5 * * * * ?")
+                .shardingItemParameters("0=A,1=B")
+                .build());
+
+    try {
+      bootstrap.schedule();
+      assertThat(job.awaitExecution(10, TimeUnit.SECONDS))
+          .as("Job should execute within timeout")
+          .isTrue();
+      await().atMost(Duration.ofSeconds(2)).until(() -> testing.spans().size() >= 2);
+    } finally {
+      bootstrap.shutdown();
+    }
+
+    List<SpanData> spans = testing.spans();
+    assertThat(spans).hasSize(2);
+
+    for (SpanData span : spans) {
+      assertThat(span.getName()).isEqualTo("SynchronizedTestSimpleJob.execute");
+      assertThat(span.getKind()).isEqualTo(SpanKind.INTERNAL);
+      assertThat(span.getAttributes().get(AttributeKey.stringKey("job.system")))
+          .isEqualTo("elasticjob");
+      assertThat(
+              span.getAttributes()
+                  .get(AttributeKey.stringKey("scheduling.apache-elasticjob.job.name")))
+          .isEqualTo("simpleElasticJob");
+      assertThat(
+              span.getAttributes()
+                  .get(AttributeKey.longKey("scheduling.apache-elasticjob.sharding.total.count")))
+          .isEqualTo(2L);
+      assertThat(span.getAttributes().get(AttributeKey.stringKey("code.function")))
+          .isEqualTo("execute");
+      assertThat(span.getAttributes().get(AttributeKey.stringKey("code.namespace")))
```

Comment:
Could do static imports for attribute keys

```suggestion
      assertThat(span.getAttributes().get(stringKey("code.namespace")))
```